### PR TITLE
Update `slugify` and add more flags

### PR DESCRIPTION
**Pull request by sunknudsen**
I also updated `@sindresorhus/slugify` to `0.11.0`.

FYI, I didn’t increment the semantic versioning, but I guess this would be `1.2.0`.

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -8,17 +8,38 @@ const cli = meow(`
 	  $ slugify <string>
 
 	Options
-	  --separator=<string>  Word separator [Default: -]
+	  --separator=<string>                    Word separator [Default: -]
+	  --lowercase=<string>                    Make the slug lowercase. [Default: true]
+	  --decamelize=<string>                   Convert camelCase to separate words [Default: true]
+	  --preserve-leading-underscore=<string>  If your string starts with an underscore, it will be preserved in the slugified string. [Default: false]
```
Comment (sindresorhus) [inline cli.js]: You need to update the readme output too.

File: cli.js [hunk 2]
```diff
@@ -8,17 +8,38 @@ const cli = meow(`
 	  $ slugify <string>
 
 	Options
-	  --separator=<string>  Word separator [Default: -]
+	  --separator=<string>                    Word separator [Default: -]
+	  --lowercase=<string>                    Make the slug lowercase. [Default: true]
```
Comment (sindresorhus) [inline cli.js]: Inconsistent using of trailing `.`

File: cli.js [hunk 3]
```diff
@@ -8,17 +8,38 @@ const cli = meow(`
...
 	  $ slugify 'Déjà Vu!'
 	  deja-vu
-	  $ slugify 'Unicorns & Rainbows' --separator='_'
+	  $ slugify 'Unicorns & Rainbows' --separator=_
 	  unicorns_and_rainbows
+	  $ slugify 'Déjà Vu!' --lowercase=false
+	  Deja-Vu
+	  $ slugify 'fooBar' --decamelize=false
+	  foobar
+	  $ slugify '_foo_bar' --preserve-leading-underscore=true
```
Comment (sindresorhus) [inline cli.js]: Drop this one. We don't need examples for every option. This one is clear enough from the description.

File: cli.js (hunk 1 above)
Comment (sindresorhus) [reply cli.js]: Also, all of these except `separator` are boolean arguments and should not have a value.

Comment (sunknudsen) [reply cli.js]: How would you like to address this given we need to set them to false (default being true for `lowercase` and `decamelize`?

`--disable-lowercase`, `--disable-decamelize` and `--preserve-leading-underscore`?

Comment (sindresorhus) [reply cli.js]: `--no-lowercase`. `meow` already handles this convention automatically.

**Pull request merged**